docs/list.js

```diff
@@ -90,6 +89,19 @@ class List {
     push(...object) {}
 
     /**
+     * Changes the contents of the list by removing objects and/or inserting new objects.
+     * @see {@linkcode https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice Array.prototype.splice}
+     * @param {number} index - The start index. If greater than the length of the list,
+     *   the start index will be set to the length instead. If negative, then the start index
+     *   will be counted from the end of the list (e.g. `list.length - index`).
+     * @param {number} [count=length - start] - The number of objects to remove from the list.
```

Comment:
Maybe should be `count=length - index`? Or change the working in the last line to be consistent.